lesson6_task1.py

```diff
@@ -0,0 +1,15 @@
+"""
+The greatest number
+
+Write a Python program to get the largest number from a list of random numbers with the length of 10
+Constraints: use only while loop and random module to generate numbers
+"""
+from random import randint
+
+my_list = []
+
+while len(my_list) < 10:
+    my_list.append(randint(1, 100))
+    if len(my_list) == 10:
```

Comment:
Also pretty nice, but I think condition `if len(my_list) == 10: ` may be omitted and you could simply print max value of your list out of while loop.